Client/Frontend/Home/HomeTabBanner.swift

```diff
@@ -0,0 +1,240 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0
+
+import Storage
+import Shared
+import UIKit
+
+/// The Home Tab Banner is the card that appears at the top of the FIrefox Homepage.
+/// 
+/// The HomeTabBanner is one UI surface that is being targeted for experimentation with `GleanPlumb` AKA Messaging.
+/// When there are GleanPlumbMessages, the card will get populated with that data. Otherwise, we'll continue showing the
+/// default browser message AKA the evergreen.
+
+class HomeTabBanner: UIView, GleanPlumbMessageManagable {
+    
+    struct UX {
+        static let cardSize = CGSize(width: 360, height: 224)
+        static let logoSize = CGSize(width: 64, height: 64)
+        static let learnHowButtonSize: CGSize = CGSize(width: 304, height: 44)
+    }
+    
+    // MARK: - Properties
+
+    public var dismissClosure: (() -> Void)?
+    var message: GleanPlumbMessage?
+    
+    // UI
+    private lazy var bannerTitle: UILabel = .build { label in
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.font = UIFont.systemFont(ofSize: 20, weight: .bold)
+        label.textColor = UIColor.theme.defaultBrowserCard.textColor
+    }
+
+    private lazy var descriptionText: UILabel = .build { label in
+        label.numberOfLines = 4
+        label.lineBreakMode = .byWordWrapping
+        label.adjustsFontSizeToFitWidth = true
+        label.font = UIFont.systemFont(ofSize: 15, weight: .regular)
+        label.textColor = UIColor.theme.defaultBrowserCard.textColor
+    }
+
+    private lazy var CTAButton: UIButton = .build { [weak self] button in
```

Comment:
Nit: rename to `ctaButton`. Naming conventions help distinguish between the class and an instance of a class.